lib/diamond/filters.ts

```diff
@@ -0,0 +1,57 @@
+import { AddressZero } from '@ethersproject/constants';
+
+export interface FacetFilter {
+  contract: string;
+  selectors: string[];
+}
```

Comment:
It looks like these `FacetFilter` objects are used in sets of 3 arrays, corresponding to the `FacetCutAction` types.  Would it not be better to include the action in the `FacetFilter` type?

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/158/commits/54da442a2170c070c2403b7f395faf573b038120